### ISSUE TITLE
Error in wkt.dumps

### ISSUE THREAD

**Issue by vadikko2** [geomet/geomet#72]
There is bug in `wkt._round_and_pad` while rounding exponential representation of coordinates with values less than e-4. Float numbers with degree less, than `10^-4` have an exponential representation. Such representation does not includes dots. Therefore `split('.')` returns empty list.

I propose to fix follow strings:

1. https://github.com/geomet/geomet/blob/8d57610226ea93a2ee0c490e5109d5778d9fd684/geomet/wkt.py#L198
```python
- rounded = repr(round(value, decimals))
+if repr(value).find('e-') == 1:
+    rounded = format(value, '.{}f'.format(decimals))
+else:
+    rounded = repr(round(value, decimals))
```
2. Update [`test_cases`](https://github.com/geomet/geomet/blob/8d57610226ea93a2ee0c490e5109d5778d9fd684/geomet/tests/wkt_test.py#L1230)  in `TestRoundAndPad.test` by adding `[(6e-6, 16), '0.0000060000000000']`.

**Comment by larsbutler** [comment]
@vadikko2 Thanks for flagging this. It does seem there is a bug with how `wkt._round_and_pad` handles numbers represented in scientific notation. I think your pull request is a good start, but I think we need to look at more edge cases. I'll play around with a few things and add some comments to your PR.

**Comment by larsbutler** [comment]
Solved by #73.
